front/src/TurtleBar/TurtleEnableButton.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import { buttonColor } from "../theme";
```

Comment:
we access colors only in `sc` components.
`import ../theme.js`

Reply:
donè